### PR TITLE
Configurable AUP and privacy notice

### LINKED ISSUE #285: Update to new EGI AUP
EGI approved a new AUP effective from 01/06/2020. The GOCDB AUP should be updated to reflect this change -
https://documents.egi.eu/public/ShowDocument?docid=3600

### PR DESCRIPTION
**Pull request by ineilson**
Closes #285

### CODE REVIEW

**Comment by gregcorbett** [comment]
Ah the landing page and the portal have different CSS files... and images... ([landing](https://goc.egi.eu/eosc-hub-v-web_150.png) vs [portal](https://goc.egi.eu/portal/img/eosc-hub-v-web_150.png) for example)

**Comment by gregcorbett** [comment]
Could the [existing icon](https://github.com/GOCDB/gocdb/blob/dev/htdocs/web_portal/img/new_window.png) be reused?

**Comment by ineilson** [comment]
Requires httpd configuration (gocdbssl.conf) alias to images directory-

`alias /images [install dir]/htdocs/images`

File: config/local_info.xml [hunk 1]
```diff
@@ -10,6 +10,14 @@
              data by all users. This can be used for a read-only failover instance. -->
     <read_only>false</read_only>
 
+    <!-- Specify the URL and description of the Acceptable Use Policy -->
+    <aup>https://documents.egi.eu/public/RetrieveFile?docid=3600&amp;filename=EGI-SPG-AUP-V3.pdf&amp;version=3</aup>
+    <aup_title>EGI Acceptable Use Policy</aup_title>
```
Comment (gregcorbett) [inline config/local_info.xml:15]: Can we generalise this for the repository?

```suggestion
    <aup>AUP location</aup>
    <aup_title>AUP title</aup_title>
```

File: config/local_info.xml [hunk 2]
```diff
@@ -10,6 +10,14 @@
...
+
+    <!-- Specify the URL and description of the Privacy Notice -->
+    <privacy_notice>/GOCDB_Privacy_Notice_20180809.pdf</privacy_notice>
+    <privacy_notice_title>GOCDB Privacy Notice</privacy_notice_title>
```
Comment (gregcorbett) [inline config/local_info.xml:19]: Can we generalise this for the repository?

```suggestion
    <privacy_notice>PN location</privacy_notice>
    <privacy_notice_title>PN title</privacy_notice_title>
```

**Comment by gregcorbett** [comment]
Can you remove [htdocs/landing/GOCDB_Privacy_Notice_20180809.pdf](https://github.com/GOCDB/gocdb/pull/290/files#diff-052d385b19228959ec4302d4d2f19bb18df1f4c5c80bc6ea26e5c2e2e21e7d72) from the PR?

**Review by gregcorbett — APPROVED** [review]

**Pull request merged**